_blog/2018/soft-multitenancy.md

```diff
@@ -0,0 +1,316 @@
+---
+title: "Istio Soft Multitenancy Support"
+overview: Using Kubernetes namespace and RBAC to create an Istio soft multitenancy environment
+publish_date: March 21, 2018
```

Comment:
Don't forget to update before publication.

Reply:
Done